### ISSUE TITLE
Can't import the module

### ISSUE THREAD

**Issue by Enchufadoo** [radical-dreamers/animated-vue#9]
Hi, I followed the instructions in the repo

`npm install --save animated-vue`

But when I run webpack I get this:

```

module.js:472 Uncaught Error: Cannot find module 'animated-vue'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (C:\Users\ariel\Proyectos\electron-youtube\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at eval (webpack-internal:///42:1:18)
    at Object.<anonymous> (http://localhost:9080/renderer.js:982:1)
    at __webpack_require__ (http://localhost:9080/renderer.js:661:30)
    at fn (http://localhost:9080/renderer.js:87:20)
    at eval (webpack-internal:///15:4:71)
webpack-internal:///1:7884 You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html
```

I'm running webpack 2.7.0 + Windows 7 64

**Comment by srodriki** [comment]
@Enchufadoo Reaaaaaally sorry for this! 

It seems when I published 0.5.2 to npm, I still had my old .npmignore file ignoring the dist folder, which now contains the main file for the package. Please update to 0.5.3 and, again, I apologize for the inconvenience!

**Comment by Enchufadoo** [comment]
dude no worries, thanks for the library @srodriki
